services/system/Accounts/ui/src/AccountSelection.tsx

```diff
@@ -550,7 +525,7 @@ export const AccountSelection = () => {
                   ? "Loading..."
                   : isInvite
                   ? `Select an account to accept invite to `
-                  : `Select an account to login to `}
+                  : `Select an account to login to  `}
```

Comment:
nit, but if I'm not mistaken, the verb form of login is "log in," so there should be a space.